lib/bsigner.js

```diff
@@ -2,18 +2,19 @@
 
 // import stuff from src/app to export
 
-const {Hardware} = require('../src/hardware');
-const {Path} = require('../src/path');
+const common = require('./common');
+const DeviceManager = require('./device/manager');
+const {Path} = require('./path');
 const {
   prepareSign,
   generateToken,
   prepareSignMultisig,
   guessPath,
   getKnownPaths
-} = require('../src/app');
+} = require('./app');
 
 // classes
-exports.Hardware = Hardware;
+exports.DeviceManager = DeviceManager;
```

Comment:
this breaks `examples/events.js`:

https://github.com/bcoin-org/bsigner/blob/1b7af216136bd6328cd0fcd134b4b116ba8b22de/examples/events.js#L3